### PR TITLE
Align tests toml

### PR DESCRIPTION
**Pull request by wolf99**
Erik has kindly added the `tests.toml` files for each exercise in #487. As per Erik's PR message, the added files assume that all tests are implemented and up to date

I am adjusting those files to align with the reality of the tests.

In cases where a test is unimplemented, I have commented the related line in the `tests.toml` rather than removing it. This is intended to make it easier to add it back again if the missing test is added soon after.

In cases where a test is implemented but is an old version I take the same above approach on the basis that the test _as currently specified_ is not present (as advised by @sleeplessByte https://exercism-team.slack.com/archives/CAQP7JL3T/p1602446417053400?thread_ts=1602446259.052700&cid=CAQP7JL3T). I am being fairly strict here, applying this even if only the test name needs adjusting.

Tests that are not expected to be implemented (e.g. are not relevant in C) are set to `false`

This does NOT account for tests that are implemented but have been _removed_ from the specification. These will need to be identified and potentially removed at a later stage.

### CODE REVIEW

**Comment by wolf99** [comment]
Allergies... man allergies. I'll have to skip allergies and come back to it.

File: exercises/grains/.meta/tests.toml [hunk 1]
```diff
@@ -0,0 +1,35 @@
...
+
+# Square raises an exception for negative squares
+"61974483-eeb2-465e-be54-ca5dde366453" = true
+
+# Square raises an exception for greater than 64 squares
+"a95e4374-f32c-45a7-a10d-ffec475c012f" = true
+
+# Total returns the total number of grains on the board
+"6eb07385-3659-4b45-a6be-9dc474222750" = true
+
```
Comment (ee7) [inline exercises/grains/.meta/tests.toml:35]: Replaced with what `canonical_data_syncer` would have output:

```suggestion
[canonical-tests]

# 1
"9fbde8de-36b2-49de-baf2-cd42d6f28405" = true

# 2
"ee1f30c2-01d8-4298-b25d-c677331b5e6d" = true

# 3
"10f45584-2fc3-4875-8ec6-666065d1163b" = true

# 4
"a7cbe01b-36f4-4601-b053-c5f6ae055170" = true

# 16
"c50acc89-8535-44e4-918f-b848ad2817d4" = true

# 32
"acd81b46-c2ad-4951-b848-80d15ed5a04f" = true

# 64
"c73b470a-5efb-4d53-9ac6-c5f6487f227b" = true

# square 0 raises an exception
"1d47d832-3e85-4974-9466-5bd35af484e3" = true

# negative square raises an exception
"61974483-eeb2-465e-be54-ca5dde366453" = true

# square greater than 64 raises an exception
"a95e4374-f32c-45a7-a10d-ffec475c012f" = true

# returns the total number of grains on the board
"6eb07385-3659-4b45-a6be-9dc474222750" = true
```

Comment (wolf99) [reply exercises/grains/.meta/tests.toml:35]: Yeesh. But OK, changes to make better names should happen in the spec, as you suggest on #1674

**Comment by wolf99** [comment]
Linked-list does not have a `canonical-data.json`, therefore there is no `tests.toml` for the exercise.
Square-root is not a canonical exercise, therefore there is no `tests.toml` for the exercise (#1582 will add this).

**Comment by wolf99** [comment]
This is ready for review. I'll take allergies on a different PR so as to keep this one just about the `tests.toml` files

**Comment by SleeplessByte** [comment]
🌊 you're on a roll!

**Pull request merged**